recipes/ptmcmcsampler/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "ptmcmcsampler" %}
+{% set version = "2.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ptmcmcsampler-{{ version }}.tar.gz
+  sha256: 1f96ea651a3c3964a7e37ca3c3924aff83a552d0b278bfcc991af265d8739ae3
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy >=1.16.3
+    - python >=3.6
+    - scipy >=1.2.0
+    - mpi4py >=3.0.3
+
+test:
+  imports:
+    - PTMCMCSampler
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jellis18/PTMCMCSampler
+  summary: Parallel tempering MCMC sampler written in Python
+  license: MIT
+  license_file: LICENSE
```

Comment:
Would be good to add a comment stating that the local copy of the license (in conda-forge) can be removed on the next release.

Reply:
Should this comment go in `meta.yaml`?

Reply:
Yeah, I think that's fine. Otherwise, someone has to remember to open an issue when the recipe gets merged or to just remember to do it the next time the version bumps.

Reply:
Comment added.